main-service/src/main/java/ru/practicum/mainservice/comment/service/EntityFinderService.java

```diff
@@ -0,0 +1,35 @@
+package ru.practicum.mainservice.comment.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.practicum.mainservice.comment.model.Comment;
+import ru.practicum.mainservice.comment.repository.CommentRepository;
+import ru.practicum.mainservice.event.model.Event;
+import ru.practicum.mainservice.event.repository.EventRepository;
+import ru.practicum.mainservice.exception.exception.NotFoundException;
+import ru.practicum.mainservice.user.model.User;
+import ru.practicum.mainservice.user.repository.UserRepository;
+
+@Service
+@RequiredArgsConstructor
+public class EntityFinderService {
+
+    private final UserRepository userRepository;
+    private final EventRepository eventRepository;
+    private final CommentRepository commentRepository;
+
+    public User findUserById(Long userId) {
+        return userRepository.findById(userId)
+                .orElseThrow(() -> new NotFoundException("User with id: " + userId + " not found."));
```

Comment:
Можно выделить повторяющийся паттерн в константу:
`private static final String EXCEPTION_TEMPLATE = "%s with id: %d not found.";`
А в исключении использовать `String. format()`:
```suggestion
               .orElseThrow(() -> new NotFoundException(String.format(EXCEPTION_TEMPLATE, "User", userId)));
```
Или даже выделить формирование строки в отдельный метод:

```
    private String getNotFoundMessage(String entity, long userId) {
        return String.format(EXCEPTION_TEMPLATE, entity, userId);
    }
 ```

Reply:
у меня в утилах уже лежали методы для этого (я про них забыл -_-")